app/src/main/java/cm/aptoide/pt/ApplicationModule.java

```diff
@@ -1126,7 +1126,7 @@ AnalyticsBodyInterceptorV7 provideAnalyticsBodyInterceptorV7(
   }
 
   @Singleton @Provides WalletService.ServiceV8 providesWalletServiceV8(
-      @Named("retrofit-wallet") Retrofit retrofit) {
+      @Named("retrofit-v7-secondary") Retrofit retrofit) {
```

Comment:
If this is a v7 service, should change WalletService.ServiceV8 to ServiceV7 and the provides method name aswell. Later we should move this to the dataprovider like we have talked.